### PR TITLE
Modifying cards styles of integrations with the most popular frameworks

### PR DESCRIPTION
**Pull request by nelsoncode019**
@patrick91 I made the changes indicated in the pull request above

### CODE REVIEW

**Comment by nelsoncode019** [comment]
eady you should see the change with the framework text centered

File: components/icons/aiohttp.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const AioHttpIcon = () => {
+  return (
+    <div className="flex justify-center items-center icon-integration">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        id="svg8"
```
Comment (marcoacierno) [inline components/icons/aiohttp.tsx:6]:
```suggestion
```

I don't think we need those attributes

File: components/icons/aiohttp.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        width={120}
+        height={120}
+        version="1.1"
```
Comment (marcoacierno) [inline components/icons/aiohttp.tsx:9]:
```suggestion
```

File: components/icons/django.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+export const DjangoIcon = () => {
+  return (
+    <div className="flex justify-center items-center">
+      <svg
+        width="120"
+        height="120"
+        viewBox="0 0 256 256"
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+        xmlnsXlink="http://www.w3.org/1999/xlink"
```
Comment (marcoacierno) [inline components/icons/django.tsx:10]: same as above
```suggestion
```

File: components/icons/fastapi.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+export const FastApiIcon = () => {
+  return (
+    <div className="flex justify-center items-center  icon-integration">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        xmlnsXlink="http://www.w3.org/1999/xlink"
```
Comment (marcoacierno) [inline components/icons/fastapi.tsx:6]: same as above

File: components/icons/fastapi.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        aria-hidden="true"
```
Comment (marcoacierno) [inline components/icons/fastapi.tsx:7]: maybe we should add this to all icons? I am not sure accessibility-wise what makes more sense

File: components/icons/flask.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+export const FlaskIcon = () => {
+  return (
+    <div className="flex justify-center items-center icon-integration">
+      {/*?xml version="1.0" encoding="UTF-8" standalone="no"?*/}
```
Comment (marcoacierno) [inline components/icons/flask.tsx:4]:
```suggestion
```

File: components/icons/flask.tsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+      <svg
+        width="130"
+        height="130"
+        viewBox="0 0 256 329"
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+        xmlnsXlink="http://www.w3.org/1999/xlink"
```
Comment (marcoacierno) [inline components/icons/flask.tsx:11]:
```suggestion
```

File: components/icons/logo-white.svg [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 600">
```
Comment (marcoacierno) [inline components/icons/logo-white.svg:1]: do you use this file?

File: components/icons/pydantic.tsx [hunk 9]
```diff
@@ -0,0 +1,14 @@
+export const PydanticIcon = () => {
+  return (
+    <div className="flex justify-center items-center icon-integration">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
```
Comment (marcoacierno) [inline components/icons/pydantic.tsx:5]:
```suggestion
```

File: components/icons/sanic.tsx [hunk 10]
```diff
@@ -0,0 +1,33 @@
+export const SanicIcon = () => {
+  return (
+    <div className="flex justify-center items-center icon-integration">
+      <svg
+        id="svg"
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+        xmlnsXlink="http://www.w3.org/1999/xlink"
```
Comment (marcoacierno) [inline components/icons/sanic.tsx:8]:
```suggestion
```

File: components/icons/sanic.tsx [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+        width={120}
+        height={120}
+        viewBox="0, 0, 400,97.00000000000001"
```
Comment (marcoacierno) [inline components/icons/sanic.tsx:11]: `00000000000000001` :D

File: components/icons/sanic.tsx [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+      >
+        <g id="svgg">
+          <path
+            id="path0"
```
Comment (marcoacierno) [inline components/icons/sanic.tsx:15]:
```suggestion
```

File: components/icons/sanic.tsx [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+            stroke="none"
+            fillRule="evenodd"
+          />
+          <path id="path2" stroke="none" fillRule="evenodd" />
+          <path id="path3" stroke="none" fillRule="evenodd" />
+          <path id="path4" stroke="none" fillRule="evenodd" />
```
Comment (marcoacierno) [inline components/icons/sanic.tsx:28]: is this needed?

File: pages/index.tsx [hunk 14]
```diff
@@ -22,13 +23,10 @@ const HomePage: NextPage<Props> = ({ version }) => (
       <p className="mt-12 mb-6 font-bold text-xl">
         Learn how to create GraphQL API using Strawberry
       </p>
-
```
Comment (marcoacierno) [inline pages/index.tsx:25]: why the removed space?

File: styles/integrations.css [hunk 15]
```diff
@@ -0,0 +1,8 @@
+.icon-integration {
```
Comment (marcoacierno) [inline styles/integrations.css:1]: nitpick: I wonder if there is a better way to do it without custom css

File: components/icons/aiohttp.tsx [hunk 16]
```diff
@@ -0,0 +1,19 @@
...
+          className="a"
```
Comment (marcoacierno) [inline components/icons/aiohttp.tsx:14]:
```suggestion
```

**Review by marcoacierno — COMMENTED** [review]
some nitpick comments

I would cleanup the svg, especially avoiding ids/classnames

File: styles/integrations.css (hunk 15 above)
Comment (nelsoncode019) [reply styles/integrations.css:1]: > nitpick: I wonder if there is a better way to do it without custom css

I don't know, but I really like to write my own styles :)

**Pull request closed without merging**